ansible/roles/jitsi-videobridge/files/jvb-health-check-oracle.sh

```diff
@@ -157,6 +157,13 @@ EOM
                   $CLEANUP_ROUTE53_DNS
               fi
 
+              echo "Dump pre-terminate stats for JVB"
+              # this script is run from different users, e.g. jsidecar, ubuntu, root, and should not use sudo commands
+              PRE_TERMINATE_STATS="/usr/local/bin/dump-pre-terminate-stats-jvb.sh"
+              if [ -f "$PRE_TERMINATE_STATS" ]; then
```

Comment:
Should this be `-x` instead of `-f` since you're about to execute it? I guess in practice it doesn't matter

Reply:
excellent point